environments/production/public.yml

```diff
@@ -62,50 +62,9 @@ pgbouncer_pool_timeout: 1
 pgbouncer_pool_mode: transaction
 pg_query_stats: True
```

Comment:
Should these other pg/pgbouner config options get moved over?

Reply:
I think I do eventually plan to move these postgresql-related config options into `postgresql.yml`, but I wanted to focus on the most challenging part first, and then all move the smaller things into place once that's settled. I have a couple more things in mind before I do that, so I'll do leave it for a followup PR if that's okay.